### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by victoria-cherkas**
* Removes the unneeded README sections which came with the SEN blueprint.
* Renames the title to reflect the new repo name.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,63 +38,6 @@ pytest
 If the tests pass, you are good to go. If not, contact the package administrator Tobias Wicky. Make sure to update your requirement files and export your environments after installation
 every time you add new imports while developing. Check the next section to find some guidance on the development process if you are new to Python and/or APN.
```
Comment (ninaburg) [inline README.md:39]:
```suggestion
every time you add new imports while developing. Check the next section to find some guidance on the development process if you are new to Python and/or SEN.
```

**Review by ninaburg — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -38,63 +38,6 @@ pytest
 If the tests pass, you are good to go. If not, contact the package administrator Tobias Wicky. Make sure to update your requirement files and export your environments after installation
```
[the remark this answers is not in the document]
Comment (victoria-cherkas) [reply README.md:38]: Thanks for spotting. Will change!

**Review by ninaburg — DISMISSED** [review]

File: README.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
-# Icon Data Processing Incubator: Prototype for a data post-processing framework on the basis of xarray.
+# meteodata-lab: Prototype for a model data processing framework on the basis of xarray.
```
Comment (cfkanesan) [inline README.md:1]: Do we want to keep the word prototype?

Comment (ninaburg) [reply README.md:1]: We could remove it from the title but add a section below as in the [ecmwf/polytope](https://github.com/ecmwf/polytope/blob/develop/readme.md) repo:

```
> [!WARNING]
> This project is BETA and will be experimental for the forseable future. Interfaces and functionality are likely to change, and the project itself may be scrapped. DO NOT use this software in any project/software that is operational.
```

**Review by cfkanesan — APPROVED** [review]

**Review by ninaburg — APPROVED** [review]

**Pull request merged**